src/redisearch_rs/rlookup/src/lookup.rs

```diff
@@ -984,7 +984,8 @@ impl<'a> RLookup<'a> {
     }
 
     pub fn init(&mut self, spcache: Option<IndexSpecCache>) {
-        // c version used memset to zero initialize, We behave the same way in release, but add a debug assert to catch misuses.
+        // C-CODE: used memset to zero initialize,
```

Comment:
Stray change?

Reply:
As always do let me know if you rather have me revert this, but either way I would consider small things like this too small for its own PR 🤷